packages/components/utils/src/Portal/Portal.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Portal'
+type: 'component'
+status: 'beta'
+slug: /utils/portal/
+typescript: ./Portal.tsx
+---
```

Comment:
I notice that you are already using the new format of the documentation in the body of this page
and that's great!

But I think I merged some changes to the plop template while this PR is in review and now I think it would be a good idea to rebase and implement some changes to the new pages

the first one is that in the frontmatter of these pages we need to pass `v4Doc: true` and we still didn't designed where to show the status badge so I think it can be removed for now. This is how the frontmatter will look like:

```suggestion
---
title: 'Portal'
type: 'component'
slug: /utils/portal/
typescript: ./Portal.tsx
v4Doc: true
---
```

Another change is the "Props (API reference)" title
I will add the comment with the changes in that line (to be continued)

Reply:
ok, no problem